public/index.html

```diff
@@ -32,7 +32,6 @@
     />
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=5, minimum-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="twitter:image" content="https://pancakeswap.finance/hero.png" />
```

Comment:
I don't think we want to remove this. if the image doesn't exist we should find another one.
@ChefBugs  can we have a default image for the meta tag ?

Reply:
it is in chungie's paws as we speak